Simplenote/src/main/java/com/automattic/simplenote/NoteEditorFragment.java

```diff
@@ -1944,27 +1953,15 @@ public void onCheckedChanged(CompoundButton buttonView, boolean isChecked) {
             chip.setOnCloseIconClickListener(new View.OnClickListener() {
                 @Override
                 public void onClick(View view) {
-                    mTagChips.removeView(view);
```

Comment:
This call is redundant because after the tag is deleted, all the chips are updated with the new tags:
```java
 private void setChips(List<String> tags) {
        mTagPadding.setVisibility(tags.size() > 0 ? View.VISIBLE : View.GONE);
        mTagChips.setVisibility(tags.size() > 0 ? View.VISIBLE : View.GONE);
        mTagChips.setSingleSelection(true);
        mTagChips.removeAllViews();
```

So the view (chip) was deleted, but all the chips were also deleted later when the new tags arrived.

Reply:
Ah, this is true, thanks so much for clarifying that for me Danilo! 🥇